code/modules/mob/living/silicon/mommi/examine.dm

```diff
@@ -20,7 +20,7 @@
 
 	if(tool_state)
 		var/obj/item/I = tool_state
-		msg += "Its utitility claw is gripping \icon[I] [I.gender==PLURAL?"some":"a"] [I.name].\n"
+		msg += "It is is gripping \icon[I] [I.gender==PLURAL?"some":"a"] [I.name] in it's utitility claw.\n"
```

Comment:
It shouldn't need to use those checks.

Should be possible to do \a [I], since Byond would format it automagically.